mappings/InventoryID-ISO-gcode-Bibkey-Source.tsv

```diff
@@ -2069,7 +2069,7 @@ InventoryID	LanguageCode	Glottocode	BibtexKey	Source
 1395	afr	afri1274	1395_afr	gm
 1396	agj	argo1244	1396_hertzon	gm
 1397	har	hara1271	1397_hertzon	gm
-1398	din		1398_din	gm
+1398	din	dink1262	1398_din	gm
```

Comment:
this one isn't in #121, is it from some other issue that we should also be closing with this PR?